tests/jballerina-unit-test/src/test/resources/test-src/annotations/deprecation_annotation.bal

```diff
@@ -297,3 +297,51 @@ function myFunction(int i, string s) {
 function testUsingDepricatedFunction() {
     myFunction(1, "hello");
 }
+
```

Comment:
Do we have tests for anonymous records?

e.g.,
```ballerina
record {|
    # Docs for name.
    @deprecated
    string name;

    string fullName;
|} r = {name: "Jo", fullName: "Jo Doe"};
```

Reply:
I will add tests

Reply:
Updated ace7f7747a3d13682f4324ac8014da4b85b9807d